README.md

```diff
@@ -1 +1,30 @@
-# actions
\ No newline at end of file
+## Fastlane GitHub Actions
+
+The Fastlane GitHub Actions provides a set of GitHub Actions to make maintaining projects easier.
+
+## Actions
+
+- 🌏 [@github-actions/fastlane-env-reminder](fastlane-env-reminder)
+
```

Comment:
You can probably get rid of these two empty lines here (8 + 9) and below (13 + 14)

Reply:
I removed just one line to keep a description in a new line 👍